quote.py

```diff
@@ -3,48 +3,102 @@
 import random
 from halibot import HalModule
 
+sr = random.SystemRandom()
```

Comment:
...eh, you're right, but they're not specific as to what systems might not support it--while it seems rare, I suppose relying on the pseudorandom internal generator is a better choice for ultimate portability. Fixing...